.luarc.json

```diff
@@ -5,11 +5,11 @@
   },
   "workspace": {
     "library": [
-      "runtime/lua",
       "${3rd}/busted/library",
-      "${3rd}/luv/library"
+      "luv"
```

Comment:
Maybe it would be better to add a separate `emmyrc.json` for a sliding migration?

Reply:
Sure. I don't plan on merging this as-is, just a place to hold all the fixes so I can post issues upstream.

Once this gets big enough, I'll cherry pick all the changes which are LuaLS compatible.

Reply:
Still, assuming (as I do) that we'll eventually want to migrate, would it then(!) make sense to bundle a preconfigured `lsp/emmyluals.lua` as well?

Reply:
Too early to tell. EmmyLuaLs is compatible with `.luarc.json`. The main difference atm is the libraries.

Reply:
(The same applies to `lsp/luals.lua`.)

Reply:
Oh sorry I misunderstood the question. Yeah sure, we can bundle both.